src/acquisition/flusurv/flusurv_update.py

```diff
@@ -81,6 +82,8 @@
 from delphi.utils.epiweek import delta_epiweeks
 
 
+max_age_to_consider_weeks = 52
```

Comment:
use caps since its a constant
```suggestion
MAX_AGE_TO_CONSIDER_WEEKS = 52
```